assets/app/view/game/corporation.rb

```diff
@@ -274,11 +274,18 @@ def render_shares
             market_tr_props[:style][:backgroundColor] = color
             market_tr_props[:style][:color] = contrast_on(color)
           end
+        end
+
+        if player_rows.any? || @corporation.num_market_shares.positive?
+          president = @corporation.owner == @game.share_pool
```

Comment:
let's add pool? method to entities so you don't do == share_pool

Reply:
75b60a63